build.sbt

```diff
@@ -219,7 +201,8 @@ lazy val ratatoolScalacheck = project
     libraryDependencies ++= Seq(
       "org.apache.avro" % "avro" % avroVersion,
       "org.scalacheck" %% "scalacheck" % scalaCheckVersion,
-      "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion
+      "com.google.apis" % "google-api-services-bigquery" % bigqueryVersion,
```

Comment:
I would also say that this should be `provided` dependency, or a separate project. Otherwise, there is a lot of problems with dependencies and tests when bigquery client version doesn't match. 

I ended up extracting `ratatool-scalacheck` when our tests had a different library version than `compile`. Tests passed, but code didn't work properly in production.

Reply:
It could be a `provided`, but I would not pull it into a separate project solely for this reason. Most conflicts should be solvable in user land so I will just extract this into a separate issue for now

Reply:
Conflicts can be solved, but it's easily possible to miss implicit version bump in tests that happened in our case.

Reply:
👍 I have changed it to provided for now